src/methods/photos.js

```diff
@@ -61,7 +61,8 @@ export default function photos(): Object {
         username,
         query: q,
         w: width,
-        h: height
+        h: height,
+        cacheBuster: cacheBuster // Avoid ajax response caching
```

Comment:
Minor, but you can use property value shorthand here. 

So
```js
let query = {
  // ...
  cacheBuster: cacheBuster
};
```
let query = {
  // ...
  cacheBuster
};
becomes:

```js

```